frontend/apps/vft-extended/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,164 @@
+import { useAccount, useAlert } from '@gear-js/react-hooks';
+import { useState } from 'react';
+
+import { useTokenActions, useTokenQueries, useTokenEvents, useBalanceOfQuery } from '../../hooks';
+
+import { toActorId } from './helper';
+import { styles } from './styles';
+
+function isZeroAddress(address: string) {
+  return address === '0x0000000000000000000000000000000000000000';
+}
+
+function Home() {
+  const { account } = useAccount();
+  const { name, symbol, decimals, totalSupply, isLoading, refetchTotalSupply } = useTokenQueries();
+  const { mint, burn, mintPending, burnPending, transfer, transferPending } = useTokenActions();
+  const alert = useAlert();
+
+  const [transferTo, setTransferTo] = useState('');
+  const [transferValue, setTransferValue] = useState('');
+
+  // Balance Of logic
+  const [balanceAddr, setBalanceAddr] = useState('');
+  const [checkedAddr, setCheckedAddr] = useState('');
+  const [showBalance, setShowBalance] = useState(false);
+
+  useTokenEvents({
+    onMinted: () => void refetchTotalSupply?.(),
+    onBurned: () => void refetchTotalSupply?.(),
+  });
+
+  const handleMint = async () => {
+    try {
+      if (!account?.address) return alert.error('No account selected!');
+      await mint(toActorId(account.address), '1000');
+      alert.success('Mint success!');
+    } catch (e) {
+      alert.error('Error mint');
+      console.error(e);
+    }
+  };
+
+  const handleBurn = async () => {
+    try {
+      if (!account?.address) return alert.error('No account selected!');
+      await burn(toActorId(account.address), '1000');
+      alert.success('Burn success!');
+    } catch (e) {
+      alert.error('Error burn');
+      console.error(e);
+    }
+  };
+
+  const handleTransfer = async () => {
+    try {
+      if (!account?.address) return alert.error('No account selected!');
+      await transfer(toActorId(transferTo), transferValue);
+      alert.success('Transfer success!');
+      setTransferValue('');
+      setTransferTo('');
+    } catch (e) {
+      alert.error('Error transfer');
+      console.error(e);
+    }
+  };
+
+  const handleBalanceOf = () => {
+    setCheckedAddr(balanceAddr.trim());
+    setShowBalance(true);
+  };
+
+  const actorId = toActorId(checkedAddr);
+  const balanceQuery = useBalanceOfQuery(actorId);
+
+  return (
+    <div style={styles.container}>
+      <div style={{ marginBottom: 24, textAlign: 'center' }}>
```

Comment:
Semantically those are forms with submit buttons, but up to you if you want to stick with it.